tests/Integration/tests/confval/confval-menu-tree/input/index.rst

```diff
@@ -6,11 +6,14 @@ Confval menu as list
 ====================
 
 ..  confval-menu::
+    :name: confval-list
 
 Confval menu as list with more info
 ===================================
 
 ..  confval-menu::
+    :caption: Confval menu as list with more info
+    :name: confval-list-info
     :type:
```

Comment:
Do we need type and default here?

Reply:
Yes, they indicate that type and default values from the listed confvals should be displayed